src/main/java/com/sda/microblogging/service/UserService.java

```diff
@@ -52,6 +53,14 @@ public User update(@Valid @NotNull User updatedUser) {
         }
     }
 
-    // TODO find All Active Users
+    public List<User> findAllActiveUsers(){
```

Comment:
GOOD, but you have to add another method that we need to find blocked user based on the id.
EXPLANATION 
We need that we find the blocked user because we can't see and we won't let them see our post.
So based on this.
I need to show which user blocked the user and to block each other from seeing each other ..